cmake/modules/FindDAOS.cmake

```diff
@@ -32,6 +32,9 @@ set(DAOS_INCLUDE_DIRS ${DAOS_INCLUDE_DIR} ${DAOS_FS_INCLUDE_DIR})
 set(DAOS_LIBRARIES ${DAOS_LIBRARY} ${DAOS_FS_LIBRARY} ${DAOS_UNS_LIBRARY})
 
 include(FindPackageHandleStandardArgs)
+include_directories( ${PC_DAOS_INCLUDEDIR} )
+link_directories( ${PC_DAOS_LIBDIR} )
+
```

Comment:
Who fills these variables? What do they contain?

Reply:
They can be filled in on the command line to build ceph, but are not required.  This change helps cmake find the daos.h header file and the dfs libs.